### ISSUE TITLE
Android webview doesnot work if change locale to Arabic.

### ISSUE THREAD

**Issue by AhmadHegazy** [wendux/DSBridge-Android#25]
while the language is English the webview works perfectly. but if I change the language to Arabic the web view does not work.

**Comment by wendux** [comment]
Where is the error ?

**Comment by AhmadHegazy** [comment]
i did not get any error and that is the problem. i will send you a demo of my code.

**Comment by Asma-Yasin** [comment]
This behavior appears to be happening with any language that uses non-English digits. I downloaded your sample and got the same behavior. After some digging, I found out that the script you're generating has callID, so I only changed %d to %s and it's working now.

The line I changed is in DWebView class inside callHandler method:
`script = String.format("%s.returnValue(%s,%s)",BRIDGE_NAME,callID, script);`
